conductr_cli/conduct_logging.py

```diff
@@ -112,3 +113,16 @@ def raise_for_status_inc_3xx(response):
     response.raise_for_status()
     if response.status_code >= 300:
         raise HTTPError(status_codes._codes[response.status_code], response=response)
+
+
+def format_timestamp(timestamp, args):
+    date = arrow.get(timestamp)
+
+    if args.date and args.utc:
+        return date.to('UTC').strftime('%Y-%m-%dT%XZ')
+    elif args.date:
+        return date.to('local').strftime('%Y-%m-%dT%X%z')
+    elif args.utc:
+        return date.to('UTC').strftime('%XZ')
+    else:
+        return date.to('local').strftime('%X%z')
```

Comment:
Are these times localised? I thought there were formatters specifically for localised formatting... That said, I think that arrow can help there, particularly with the `humanize` method. I believe its methods are locale specific.